src/taipy/gui/gui_actions.py

```diff
@@ -195,3 +196,74 @@ def invoke_callback(gui: Gui, state_id: str, callback: t.Callable, args: t.Union
 def invoke_state_callback(gui: Gui, state_id: str, callback: t.Callable, args: t.Union[t.Tuple, t.List]) -> t.Any:
     warnings.warn("'invoke_state_callback()' was deprecated in Taipy GUI 2.0. Use 'invoke_callback()' instead.")
     return invoke_callback(gui, state_id, callback, args)
+
+
+def invoke_long_running(
+    state: State,
+    user_function: t.Callable,
+    user_function_args: t.Union[t.Tuple, t.List] = [],
+    user_status_function: t.Optional[t.Callable] = None,
+    user_status_function_args: t.Union[t.Tuple, t.List] = [],
+    delay=0,
```

Comment:
Rename to 'period'
Unit should be milliseconds for homogeneity, with a minimal value of 500 (or whatever)